### ISSUE TITLE
403 Forbidden for macos native prebuilt with Kotlin 1.6 

### ISSUE THREAD

**Issue by joffrey-bion** [LouisCAD/CompleteKotlin#9]
I'm running into this error now while trying to upgrade to Kotlin 1.6:
```
> Could not resolve :kotlin-native-prebuilt-macos-aarch64:1.6.
  Required by:
      project :autobahn-tests
   > Could not resolve :kotlin-native-prebuilt-macos-aarch64:1.6.
      > Could not get resource 'https://complete-kotlin-prebuilt.louiscad.com/kotlin/native/builds/releases/1.6/macos-aarch64/kotlin-native-prebuilt-macos-aarch64-1.6.tar.gz'.
         > Could not HEAD 'https://download-cdn.jetbrains.com/kotlin/native/builds/releases/1.6/macos-aarch64/kotlin-native-prebuilt-macos-aarch64-1.6.tar.gz'. Received status code 403 from server: Forbidden
```

Is your mirror up to date with 1.6 binaries?

**Comment by LouisCAD** [comment]
I think I need to update the code.
I'll try to find time to fix it next week.

**Comment by joffrey-bion** [comment]
Ok, thanks a lot for looking into this! If I can help in some way please let me know

**Comment by joffrey-bion** [comment]
Any progress on this?

**Comment by LouisCAD** [comment]
Hello, sorry for the radio silence, I was a bit busy and lacked the feeling of breathing room to focus on that deeply.

I'll try to update the plugin for new Kotlin versions this afternoon, but I might not be done today, we'll see.

If you want to help, we can pair-program together, you can reach me on Kotlin Slack or via Twitter DM.

**Comment by LouisCAD** [comment]
Should be fixed in version 1.1.0! (just released)

**Comment by joffrey-bion** [comment]
Sorry for not replying with an actual message in the first place, I was afk when I saw your response. Thanks for coming back to this.
I totally understand for the radio silence. This is OSS, you don't owe anyone anything 🙂

**Comment by joffrey-bion** [comment]
Note that this issue was NOT reproducible since Kotlin 1.6.10 for some reason.

But I tried again with Kotlin 1.6.0 + CompleteKotlin 1.0.0 and I could reproduce, and then upgraded to 1.1.0 and this solved the problem. Thanks again!

**Comment by LouisCAD** [comment]
It's because the native compiler versioning scheme changed after I published 1.0.0, and for non .0 releases, there was a match between the old and the new versioning schemes.

I initially wanted to support both, which made it way harder at the time, and made me postpone the completion for a while, and now, supporting older versions is not a big issue anymore, so I decided to show an error message if used on Kotlin version preceding 1.5.30.

**Comment by joffrey-bion** [comment]
Makes sense. Thanks for the additional info!
